.github/workflows/add-milestone.yml

```diff
@@ -0,0 +1,48 @@
+# Once a PR has been merged, this action adds it to the upcoming release.
+#
+# See https://github.com/google/web-stories-wp/issues/6301
+
+name: Add milestone
+
+on:
+  pull_request:
+    types:
+      - closed
+
+env:
+  PR_NUMBER: ${{ github.event.pull_request.number }}
+  ISSUE_REFERENCE_REGEX: \(#([0-9]+)\)
+  GITHUB_REPO_ID: 235435637
```

Comment:
What is this name? Should it be a secret?

Reply:
This is just the public ID of this repository as found in the GitHub API. Nothing confidential.

See https://github.com/ZenHubIO/API#endpoint-reference